src/styles/shared/_buttons.scss

```diff
@@ -37,96 +36,121 @@
   -webkit-tap-highlight-color: transparent;
 
   &:hover {
-    background: linear-gradient(
-      to bottom,
-      color('sky', 'lighter'),
-      color('sky', 'light')
+    background: var(
+      --p-interactive-neutral-elevation-3,
+      linear-gradient(to bottom, color('sky', 'lighter'), color('sky', 'light'))
     );
-    border-color: color('sky', 'dark');
+    border-color: var(--p-none, color('sky', 'dark'));
   }
 
   &:focus {
-    border-color: color('indigo');
+    border-color: var(--p-transparent, color('indigo'));
     outline: 0;
-    box-shadow: 0 0 0 1px color('indigo');
+    box-shadow: var(--p-none, 0 0 0 1px color('indigo'));
 
+    @include focus-ring;
     @include high-contrast-button-outline;
```

Comment:
@dpersing ping me when you’re back and let’s talk about resolving the new focus ring styles with the high contrast outline. I think we can achieve both in a single CSS approach and mixin.